MIGRATING.md

```diff
@@ -0,0 +1,156 @@
+# Okta Java Management SDK Migration Guide
+ 
+This SDK uses semantic versioning and follows Okta's [library version policy](https://developer.okta.com/code/library-versions/). In short, we do not make breaking changes unless the major version changes!
+
+## Migrating from 1.x.x to 2.0.0
+
+Version 2.0.0 of this SDK introduces a number of breaking changes from previous versions. 
+In addition to many new classes/interfaces, some existing classes/interfaces are no longer backward compatible due to method renaming and signature changes.
+
+#### All `*Factor` classes Renamed to `*UserFactor`
+This means that you will now have to start using the new `*UserFactor` objects instead.
+
+### Package `com.okta.sdk.client.Client`
+
+Below methods have been renamed for the sake of clarity.
+
+   - Renamed `createRule` to `createGroupRule`
+   - Renamed `getRule` to `getGroupRule`
+   - Renamed `listRules` to `listGroupRules`
+
+### Package `com.okta.sdk.resource.log.LogEventList`
+
+The API used to get log info has undergone a signature change (note the order swap of `until` and `since` parameters).
+
+   - `getLogs(String until, String since, String filter, String q, String sortOrder)`
+   Signature changed to `getLogs(Date since, Date until, String filter, String q, String sortOrder)`
+   
+### Package `com.okta.sdk.resource.group.GroupList`
+
+The API used to list the groups to which a user belongs, has undergone a signature change. 
+There is no need for supplying the `expand` parameter anymore. 
+
+   - `listGroups(String q, String filter, String expand)`
+   Signature changed to `listGroups(String q, String filter)`
+   
+With this removal of `expand` parameter, the caller has two options of achieving the previous result:
```

Comment:
This should be indented, take a look at how the page renders

Reply:
fixed